.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Configuration file for eslint, can be modified to be more/less restrictive in a later PR.